src/plugins/intel_cpu/src/nodes/random_uniform.cpp

```diff
@@ -240,19 +368,20 @@ inline void runPhilox(uint64_t key, uint64_t counter, uint64_t n, uint32_t* res)
     uint32_t* counter_32 = reinterpret_cast<uint32_t*>(&counter);
     uint32_t* n_32 = reinterpret_cast<uint32_t*>(&n);
 
-    for (size_t i = 0lu; i < ROUNDS_NUMBER; i++) {
+    for (size_t i = 0lu; i < ROUNDS_NUMBER - 1; i++) {
         calculateRound(key_32, counter_32, n_32);
-        if (i < ROUNDS_NUMBER - 1)
-            raiseKey(key_32);
+        raiseKey(key_32);
     }
+    raiseKey(key_32);
+
```

Comment:
Could you tell me please why it was changed?
Before if `i = ROUNDS_NUMBER - 1` we called only `calculateRound(key_32, counter_32, n_32)`, now only `raiseKey(key_32)`.

Reply:
Slight performance benefit, as there is no more comparisons inside the for loop.